isb/cron/generate_sitemap.sh

```diff
@@ -1,18 +1,16 @@
 #!/bin/bash
-declare ROOT=/app
-export PYTHONPATH=$ROOT
-cd $ROOT
-mkdir $ROOT/sitemaps
+export PYTHONPATH=/app
+cd /app
 mkdir /var/log/isamples
 declare DATE=$(date "+%Y-%m-%d")
-declare OUTPUT_DIR="$ROOT/$DATE/sitemaps"
+declare OUTPUT_DIR="/app/sitemaps/$DATE/sitemaps"
 echo "Attempting to remove sitemap directory output at $OUTPUT_DIR, ok if this fails"
 rm -rf $OUTPUT_DIR
 echo "Creating sitemap directory output at $OUTPUT_DIR"
 mkdir -p $OUTPUT_DIR
 echo "Invoking sitemap creation script, using sitemap prefix of $ISB_SITEMAP_PREFIX"
-/usr/local/bin/python scripts/generate_things_sitemap.py -h "$ISB_SITEMAP_PREFIX" -p "$OUTPUT_DIR"  >& /var/log/isamples/$DATE.sitemap.txt
+/usr/local/bin/python /app/scripts/generate_things_sitemap.py -h "$ISB_SITEMAP_PREFIX" -p "$OUTPUT_DIR"  >& /var/log/isamples/$DATE.sitemap.txt
 echo "Removing old sitemaps symlink"
-rm -rf $ROOT/isb_web/sitemaps
+rm -rf /app/isb_web/sitemaps
 echo "Creating new sitemaps symlink"
-ln -s $OUTPUT_DIR $ROOT/isb_web/
\ No newline at end of file
+ln -s $OUTPUT_DIR /app/isb_web/
```

Comment:
Changes to the sitemap generation script to put them in the directory where the volume is mounted